### PR TITLE
Create 2019-01-17-python-ecology-es.md

### PR DESCRIPTION
**Pull request by raynamharris**
a blog post about translating the Data Ecology Python lesson. Please wait for approval from co-authors and some people mention. I'll post a message when its okay to merge. Thanks!

### CODE REVIEW

File: _posts/2019/01/2019-01-17-python-ecology-es.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+layout: page
+authors: ["Rayna Harris", "Paula Andrea Martinez", "Heladia Salgado"]
+title: "Data Carpentry Python Ecology Lesson Translated into Spanish"
+teaser: "We are excited to announce our newest community-driven lesson translation: Análisis y visualización de datos usando Python"
+date: 2019-01-17
+time: "01:00:00"
+tags: ["Translation", "Latin America" "Lessons", "Data Carpentry", "Community"]
+---
+
+From September 2017 to May 2018, there was [community-driven effort](https://software-carpentry.org/blog/2018/03/forlatinamerica.html) to translate the Software Carpentry lessons into Spanish, and the result was the production three high-quality lessons that can be used to teach a Software Carpentry Workshop consisting of [La Terminal de Unix](https://swcarpentry.github.io/shell-novice-es/), [Control de Versiones con Git](https://swcarpentry.github.io/git-novice-es/), and [R para Análisis Científicos Reproducibles](https://swcarpentry.github.io/r-novice-gapminder-es/).
```
Comment (charlesreid1) [inline _posts/2019/01/2019-01-17-python-ecology-es.md]: should read "the production **of** three high-quality"

File: _posts/2019/01/2019-01-17-python-ecology-es.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+authors: ["Rayna Harris", "Paula Andrea Martinez", "Heladia Salgado"]
+title: "Data Carpentry Python Ecology Lesson Translated into Spanish"
+teaser: "We are excited to announce our newest community-driven lesson translation: Análisis y visualización de datos usando Python"
+date: 2019-01-17
+time: "01:00:00"
+tags: ["Translation", "Latin America" "Lessons", "Data Carpentry", "Community"]
```
Comment (charlesreid1) [inline _posts/2019/01/2019-01-17-python-ecology-es.md]: If you remove the double quotes, and change this to just `tags: [Translation, Latin America, Lessons, Data Carpentry, Community]`, the Travis CI build problem should be fixed.

**Review by charlesreid1 — COMMENTED** [review]
See comment re fixing the Travis build problem

**Review by ComplejoC — CHANGES_REQUESTED** [review]
Saludos! Pudieran añadir mi segundo apellido? Mi nombre completo es Sofía Meléndez Cartagena

File: _posts/2019/01/2019-01-17-python-ecology-es.md (hunk 1 above)
Comment (NPalopoli) [reply _posts/2019/01/2019-01-17-python-ecology-es.md]: And line 11 is missing "a" in "there was a community-driven effort"

File: _posts/2019/01/2019-01-17-python-ecology-es.md (hunk 2 above)
Comment (raynamharris) [reply _posts/2019/01/2019-01-17-python-ecology-es.md]: thank you! I've been wondering what was wrong.

Comment (raynamharris) [reply _posts/2019/01/2019-01-17-python-ecology-es.md]: done! https://github.com/carpentries/carpentries.org/pull/349/commits/9d010ccd4b425c73eeb7c9c9de1e6f44bc65f956

File: _posts/2019/01/2019-01-17-python-ecology-es.md (hunk 1 above)
Comment (raynamharris) [reply _posts/2019/01/2019-01-17-python-ecology-es.md]: Thanks! missing words added with https://github.com/carpentries/carpentries.org/pull/349/commits/c3e9dfe4376e79ce2ca47992544834ac3cd76bda

**Comment by raynamharris** [comment]
Gracias @ComplejoC! Agregué tu apellido https://github.com/carpentries/carpentries.org/pull/349/commits/80e604529fd6cd7c56f1fb94d17991dd524eb146

**Comment by raynamharris** [comment]
Ready to post @fmichonneau

**Comment by fmichonneau** [comment]
Thanks!

**Pull request merged**